projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/no-linked-cases/no-linked-cases.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { LinkedCasesService } from '../../services/linked-cases.service';
+import { NoLinkedCasesComponent } from './no-linked-cases.component';
+
+describe('NoLinkedCasesComponent', () => {
+  let component: NoLinkedCasesComponent;
+  let fixture: ComponentFixture<NoLinkedCasesComponent>;
+  let router: any;
+  let linkedCasesService: any;
+  const caseId = '1682374819203471';
+
+  router = {
```

Comment:
@johnbenjamin-hmcts - can be declared immediately at Line 11

Reply:
@Daniel-Lam I have updated the code now.